src/Microsoft.OData.Core/UriParser/Aggregation/ApplyBinder.cs

```diff
@@ -98,9 +98,9 @@ private IEdmTypeReference CreateAggregateExpressionTypeReference(SingleValueNode
                 }
             }
 
-            switch (withVerb)
+            switch (withVerb.MethodType)

```

Comment:
The change to AggregationMethod makes the name "withVerb" no longer appropriate; please change to "method".  There are a handful of place in the code where this change needs to be made: AggregateExpressionToken.cs, ApplyBinder.cs, UriQueryExpressionParser.cs, UriQueryExpressionParserTests.cs.